frontend/dashboard/components/ResourcesRepoList/ResourcesRepoList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { RESOURCEADM_BASENAME } from 'app-shared/constants';
+import { useSelectedContext } from 'dashboard/hooks/useSelectedContext';
+import { ResourceTable } from 'resourceadm/components/ResourceTable';
+import { useGetResourceListQuery } from 'resourceadm/hooks/queries/useGetResourceListQuery';
+import { getResourceDashboardURL, getResourcePageURL } from 'resourceadm/utils/urlUtils';
+import { getReposLabel } from 'dashboard/utils/repoUtils';
+import { Organization } from 'app-shared/types/Organization';
+import { useTranslation } from 'react-i18next';
+import { SelectedContextType } from 'app-shared/navigation/main-header/Header';
+import { AltinnSpinner } from 'app-shared/components';
+
+type ResourcesRepoListProps = {
+  organizations: Organization[];
+};
+
+export const ResourcesRepoList = ({ organizations }: ResourcesRepoListProps): React.ReactNode => {
+  const { t } = useTranslation();
+  const selectedContext = useSelectedContext();
+  const repo = `${selectedContext}-resources`;
+
+  const isOrganisation =
+    selectedContext !== SelectedContextType.All && selectedContext !== SelectedContextType.Self;
+  const { data: resourceListData, isLoading } = useGetResourceListQuery(
+    selectedContext,
+    !isOrganisation,
+  );
+
+  if (!isOrganisation) {
+    return null;
+  }
+
+  return (
+    <div data-testid='resource-table-wrapper'>
+      <h2>
+        {getReposLabel({
+          selectedContext,
+          orgs: organizations,
+          t,
+          isResourcesRepo: true,
+        })}
+      </h2>
+      {isLoading ? (
+        <AltinnSpinner spinnerText={t('general.loading')} />
+      ) : (
+        <div>
+          <a href={`${RESOURCEADM_BASENAME}${getResourceDashboardURL(selectedContext, repo)}`}>
```

Comment:
Here we should use the Link component from the design system instead of native anchor tag. 😊 

https://storybook.designsystemet.no/?path=/docs/felles-link--docs